src/test/java/com/neo4j/docker/neo4jserver/plugins/TestBundledPluginInstallation.java

```diff
@@ -57,10 +58,14 @@ private GenericContainer createContainerWithBundledPlugin(String pluginName)
 
     @ParameterizedTest(name = "testBundledPlugin_{0}")
     @MethodSource("bundledPluginsArgs")
-    public void testBundledPlugin(String pluginName, Neo4jVersion bundledSince, boolean isEnterpriseOnly) throws Exception
+    public void testBundledPlugin(String pluginName, Neo4jVersion bundledSince, Neo4jVersion bundledUntil, boolean isEnterpriseOnly) throws Exception
     {
         Assumptions.assumeTrue( TestSettings.NEO4J_VERSION.isAtLeastVersion( bundledSince ),
-                                String.format("plugin %s was not bundled in Neo4j %s", pluginName, bundledSince.toString()));
+                                String.format("plugin %s was not bundled in Neo4j %s", pluginName, bundledSince));
+        if(bundledUntil != null) {
+            Assumptions.assumeTrue( TestSettings.NEO4J_VERSION.isOlderThan( bundledUntil ),
+                                    String.format("plugin %s was not bundled in Neo4j until %s", pluginName, bundledUntil));
```

Comment:
I would prefer if this said `"plugin %s was not bundled after Neo4j %s"...` just because this statement makes it sound like we're testing for current version being newer than `bundledUntil` which doesn't seem to be the case with the logic here.

This is a minor issue though.